packages/clerk-js/src/v4/primitives/Input.tsx

```diff
@@ -0,0 +1,44 @@
+import { createVariants, PrimitiveProps, StyleVariants } from '../styledSystem';
+import { useInput } from './hooks/useInput';
+
+// TODO: Connect with CSS variables
+
+const { applyVariants, filterProps } = createVariants((theme, props) => ({
+  base: {
+    margin: 0,
+    outline: 0,
+    padding: `${theme.sizes.$2x5} ${theme.sizes.$4}`,
+    fontStyle: 'normal',
+    fontWeight: theme.fontWeights.$normal,
+    border: '1px solid',
+    borderColor: props.hasError ? theme.colors.$danger500 : theme.colors.$blackAlpha300,
+    borderRadius: theme.radii.$md,
+    '&:focus-visible': {
```

Comment:
We'll need to talk with Cooper about this one, but I think the focus ring should always use the primary color of the app. 

You can use `cssutils.addFocusRing` from this PR so we can update everything at once in the future: https://github.com/clerkinc/javascript/pull/263

Reply:
On the next onee